modules/miner/miner.go

```diff
@@ -33,14 +39,15 @@ type Miner struct {
 	earliestTimestamp types.Timestamp
 	address           types.UnlockHash
 
-	// A list of blocks that have been through SubmitBlock.
+	// A list of blocks that have been submitted through SubmitBlock.
 	blocksFound []types.BlockID
 
 	// BlockManager variables. The BlockManager passes out and receives unique
 	// block headers on each call, these variables help to map the received
 	// block header to the original block. The headers are passed instead of
 	// the block because a full block is 2mb and is a lot to send over http.
-	blockMem    map[types.BlockHeader]types.Block
```

Comment:
need to update the comment above these variables to better explain what they do.